rra2json.py

```diff
@@ -187,6 +187,55 @@ def autoassign_rras(config):
     except IndexError:
         debug("No unassigned RRAs")
 
+def autoassign_vas(config):
+    """This will search through unassigned RRA bugs and assign them automatically"""
+    bcfg = config['bugzilla']
+
+    # If no API key has been specified, just skip this
+    if len(bcfg['api_key']) == 0:
+        return
+
+    b = bugzilla.Bugzilla(url=bcfg['url'], api_key=bcfg['api_key'])
+
+    try:
+        with open(bcfg['cache'], 'rb') as f:
+            assign_list = pickle.load(f)
```

Comment:
We're going to need a separate round robin queue for the rotation assignments

Reply:
I'm guessing just having a name-spaced cache will do the trick.

Reply:
yeah since its a basic pickle just.. `assign_list['va']['names']` (ideal, but also fix the code for the other assign list to look consistent) or even just a separate new variable